docs/core/project-sdk/overview.md

```diff
@@ -89,35 +89,39 @@ The following table shows which element and which [globs](https://en.wikipedia.o
 > [!NOTE]
 > The `./bin` and `./obj` folders, which are represented by the `$(BaseOutputPath)` and `$(BaseIntermediateOutputPath)` MSBuild properties, are excluded from the globs by default. Excludes are represented by the property `$(DefaultItemExcludes)`.
 
-If you explicitly define these items in your project file, you're likely to get the following error:
+#### Build errors
 
-**Duplicate Compile items were included. The .NET SDK includes Compile items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file.**
+If you explicitly define any of these items in your project file, you're likely to get one or more of the following build errors:
 
-To resolve the error, either remove the explicit `Compile` items that match the implicit ones listed on the previous table, or set the `EnableDefaultCompileItems` property to `false`, which disables implicit inclusion:
+**NETSDK1022: Duplicate 'Compile' items were included. The .NET SDK includes 'Compile' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file.**
```

Comment:
You could put these in a blockquote instead of making them all bold.

Reply:
I'm going to unbold them and put them in quotes, as that seems to be the [Style Guide guidance](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36402). LMK if you know of other guidance for error messages.

Reply:
Quotation marks work well for sentence-length quotations; for longer, standalone blocks of text, blockquote is the equivalent of quotation marks.

Reply:
@tdykstra Is this what you meant?

https://review.docs.microsoft.com/en-us/dotnet/core/project-sdk/overview?branch=pr-en-us-18328#build-errors

Reply:
Yes, that's it.